docs/superpowers/plans/2026-08-06-homebrew-scoop-distribution.md

```
@@ -0,0 +1,877 @@
+brew install {{github_user}}/tap/{{github_repo_name}}
+```
+{% endif %}
+{% if include_scoop %}
+Install with [Scoop](https://scoop.sh) (Windows):
+
+```sh
+scoop bucket add {{github_user}} https://github.com/{{github_user}}/scoop-bucket
+scoop install {{github_user}}/{{github_repo_name}}
+```
+{% endif %}
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)